tutorials/buoyantSimpleFoam-aster/fluid/Allclean

```diff
@@ -0,0 +1,2 @@
+. $WM_PROJECT_DIR/bin/tools/CleanFunctions
+cleanCase
```

Comment:
Remove this file and move the functionality into the `Allclean` in the parent directory.